wpilibj/src/main/java/edu/wpi/first/wpilibj/ADIS16470_IMU.java

```diff
@@ -650,12 +657,6 @@ private void writeRegister(int reg, int val) {
     m_spi.write(buf, 2);
   }
 
-  public void reset() {
-    synchronized (this) {
-      m_integ_angle = 0.0;
-    }
-  }
```

Comment:
Is there a new way to reset the angle?

Reply:
Yes, the method was just moved to a [later portion of the file](https://github.com/wpilibsuite/allwpilib/blob/7dae805a8715ac33b94ea2205dbae5add12505c9/wpilibj/src/main/java/edu/wpi/first/wpilibj/ADIS16470_IMU.java#L968C1-L968C1). Not sure why I did that, maybe a holdover from our team's fix.